remote-api/src/main/java/org/alfresco/rest/api/Categories.java

```diff
@@ -53,6 +53,13 @@
 
     void deleteCategoryById(String id, Parameters parameters);
 
+    /**
+     *
+     * @param nodeId
+     * @return
```

Comment:
Very minimal Javadoc! Probably better to delete this if you're not going to explain what it does.

Reply:
Interfaces... so easy to miss them - I've forgot to fill that. Will be fixed in a second.